assignments/prototypes.js

```diff
@@ -32,6 +58,21 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(attrs){
+  GameObject.call(this, attrs);
+  CharacterStats.call(this, attrs);
+  this.team = attrs.team;
+  this.weapons = attrs.weapons;
+  this.language = attrs.language;
+}
+
+Humanoid.prototype = Object.create(GameObject.prototype);
+Humanoid.prototype = Object.create(CharacterStats.prototype);
```

Comment:
here as well don't need the `GameObject` prototype here. it should be inheriting that from `CharacterStats`